wiki/Beatmap_Editor/Kiai_Time/en.md

```diff
@@ -1,20 +1,24 @@
-# Kiai time
+---
+tags:
+  - timing section
+---
 
-![Stars shower the screen indicating Kiai Time has started.](img/Kiai_Time_test.jpg "Stars shower the screen indicating Kiai Time has started.")
+# Kiai Time
 
-![Kiai Time shows in the upper-left of the screen when active during edit mode.](img/Kiai_Time_edit.jpg "Kiai Time shows in the upper-left of the screen when active during edit mode.")
+*For regulations surrounding kiai time, see the [Ranking Criteria](/wiki/Ranking_Criteria).*
 
-**Kiai Time** is defined by the [mapper](/wiki/Glossary) on a [beatmap](/wiki/Beatmaps) between 2 [Timing Sections](/wiki/Beatmap_Editor/Timing). You can recognize a Kiai time because the screen flashes and you will see a star fountain, afterwards all [Hit Objects](/wiki/Hit_Objects) will flash at the rhythm of the [main beat](/wiki/Beatmap_Editor/Timing) and stars will explode from the cursor when a [Hit Circle](/wiki/Hit_Objects) is hit or will fall from the cursor when you follow a [slider](/wiki/Hit_Objects) or a [spinner](/wiki/Hit_Objects).
+**Kiai Time** is a term given for a "special" timing section to help emphasize a part of a [beatmap](/wiki/Beatmaps). A player can tell if a part of a beatmap has a kiai timing section by seeing added visual effects such as flashing beats, flying stars, and fountains.
 
-## Kiai Time on Beatmapping
+While kiai time is active all [hit objects](/wiki/Hit_Objects) will flash at the rhythm of the [main beat](/wiki/Beatmap_Editor/Timing) and stars will explode from the cursor when a [hit circle](/wiki/Hit_Objects) is hit or will fall from the cursor when a [slider](/wiki/Hit_Objects) or [spinner](/wiki/Hit_Objects) is followed.
 
-The most common term for Kiai Time when [editing](/wiki/Beatmap_Editor) beatmaps is the "strongest part of the song" this is normally the chorus of the song, however some songs don't have a chorus, but they do have a strong part for you that feels cool, so you could insert Kiai Time in that section. Be advised however that spamming kiai start sections will cause a spam in kiai fountains. That is unrankable, and mostly hated by players because of the lag it may cause.
+## Beatmapping
 
-## Things to take into consideration when using Kiai
+Kiai time is most commonly used for the "strongest part of a song" which usually is the chorus. Since kiai sections are the strongest part of a song it makes sense that they are slightly more challenging than the rest of the mapset to follow the song.
```

Comment:
```suggestion
Kiai time is most commonly used for the "strongest part of a song" which is usually the chorus. Since kiai sections are the strongest part of a song, they are typically slightly more challenging than the rest of the [beatmapset](/wiki/Beatmapsets) to follow the song.
```